iOS Client/TabController/AppCoordinator.swift

```diff
@@ -43,6 +44,51 @@ class AppCoordinator: NSObject, Coordinator, UITabBarControllerDelegate, StoreSu
             self.addChildCoordinator(childCoordinator: coord)
             return coord.rootViewController
         })
+        promptForNotifications()
+    }
+    
+    func promptForNotifications() {
+        let center = UNUserNotificationCenter.current()
+        
+        center.requestAuthorization(options: [.alert, .badge, .sound]) { granted, error in
+            if granted {
+                MBStore().syncDevotions { devotions, error in
+                    if error != nil {
+                        print("Could not sync devotions and schedule devotions")
+                        print(error)
+                    } else if devotions != nil {
+                        center.delegate = self
+                        self.scheduleNotifications(devotions: devotions!)
+                    }
+                }
+            } else {
+                print(error)
+            }
+        }
+    }
+    
+    func scheduleNotifications(devotions: [MBDevotion]) {
+        print("scheduling notifications")
+        let center = UNUserNotificationCenter.current()
+        center.removeAllPendingNotificationRequests()
+        devotions.forEach { devotion in
+            guard let devotionDay = Formatters.devotionDateFormatter.date(from: devotion.date), let calendar = Formatters.calendar else {
+                return
```

Comment:
probably want `continue` so we don't miss out on the ones after this.

Honestly, might be worth crashing if there's something wrong, especially if we just bundle in the devotional data b/c then we can catch any issues before release